### PR TITLE
docs: Adjust getting started flow

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

**Comment by jeromelaban** [comment]
@unodevops update

**Comment by jeromelaban** [comment]
@unodevops update

**Comment by jeromelaban** [comment]
@unodevops update

**Comment by jeromelaban** [comment]
@unodevops update

File: doc/articles/additional-windows-setup-inline.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+uid: Uno.GetStarted.AdditionalSetup.windows
+---
+
+In order to run Skia+GTK heads, you will need to make sure to install the [GTK3 runtime](https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases).
```
Comment (carldebilly) [inline doc/articles/additional-windows-setup-inline.md:5]: We should use "aka" for such links.

File: doc/articles/get-started-rider.md [hunk 2]
```diff
@@ -16,85 +18,34 @@ uid: Uno.GetStarted.Rider
...
-Creates a new project:  
-```bash
-dotnet new unoapp -o MyApp
-```
+<details>
+    <summary>† Notes (Click to expand)</summary>
 
-You should now have a folder structure that looks like this:  
-![rider-folder-structure](Assets/quick-start/rider-folder-structure.JPG)
+  * **WebAssembly**: debugging from the IDE is not available yet on Rider.  But you can use the [Chromium in-browser debugger](https://github.com/unoplatform/Uno.Wasm.Bootstrap#how-to-use-the-browser-debugger) instead.
```
Comment (carldebilly) [inline doc/articles/get-started-rider.md:24]: This link doesn't exists anymore.

File: doc/articles/get-started-wizard.md [hunk 3]
```diff
@@ -6,156 +6,15 @@ uid: Uno.GetStarted.Wizard
...
 **Congratulations, you've just created a new project using the [Uno Platform](https://platform.uno/) project templates!**
 
-[!include[getting-help](use-uno-check-inline.md)]
-
 ## Next steps
 
 * You can give the [Single Page app getting started guide](https://platform.uno/docs/articles/getting-started-tutorial-1.html) a try
 * Next you can try our [Bug Tracker sample walkthrough](https://platform.uno/docs/articles/getting-started-tutorial-2.html)
 * More advanced examples in our [Uno.Samples repository](https://github.com/unoplatform/uno.samples)
 * Fork a fully-fledged [Ch9 application and source code ](https://platform.uno/code-samples/#ch9)
```
Comment (carldebilly) [inline doc/articles/get-started-wizard.md:14]: We should refer to page ids instead.

**Review by carldebilly — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.0

**Pull request merged**